compiler/rustc_metadata/src/native_libs.rs

```diff
@@ -385,7 +385,11 @@ impl<'tcx> Collector<'tcx> {
                             let link_ordinal_attr = self
                                 .tcx
                                 .hir()
-                                .attrs(self.tcx.hir().local_def_id_to_hir_id(child_item.id.def_id))
+                                .attrs(
+                                    self.tcx
+                                        .hir()
+                                        .local_def_id_to_hir_id(child_item.id.def_id.def_id),
```

Comment:
Since we have an `OwnerId`, we statically know that `local_id` is 0. Can we get an `impl From<OwnerId> for HirId` for this case?